resources/patches/core-locale-prevent-remote-loading.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/multisiteplayground_master_production.2.1/includes/locale.inc b/multisiteplayground_master_production.2.1/includes/locale.inc
+index 82c55e5..e086f0d 100644
+--- includes/locale.inc
++++ includes/locale.inc
+@@ -1485,6 +1485,10 @@ function _locale_parse_js_file($filepath) {
+   $parsed_url = drupal_parse_url($filepath);
+   $filepath = $parsed_url['path'];
+   // Load the JavaScript file.
```

Comment:
Here we make sure file_get_contents never gets remote URI as it would bypass the drupal_http_request method.

This happens when someone uses remote uri with drupal_add_js but forgets to set the type as external.

The locale module then tries to get its content anyway.